src/components/auth/InputField/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+
+import { ErrorMessage } from '@hookform/error-message';
+import classNames from 'classnames/bind';
+import { useFormContext } from 'react-hook-form';
+
+import { PASSWORD_SHOW_MODE } from '@/constants';
+
+import useToggleButton from '@/hooks/useToggleButton';
+
+import styles from './InputField.module.scss';
+
+const cx = classNames.bind(styles);
+
+type InputFieldProps = {
+  label: string;
+  name: string;
+  type?: 'text' | 'email' | 'password';
+  placeholder?: string;
+  maxLength?: number;
+};
+
+const InputField = ({ label, name, type = 'text', ...props }: InputFieldProps) => {
```

Comment:
```suggestion
type SignInputFieldProps = {
  label: string;
  name: string;
  type?: 'text' | 'email' | 'password';
  placeholder?: string;
  maxLength?: number;
};

const SignInputField = ({ label, name, type = 'text', ...props }: SignInputFieldProps) => {
```
공용 컴포넌트인 InputField와 구별하기 위해서 SignInputField로 바꾸면 어떨까요?